src/components/Tool/Appeal/List/AppealsListFilterPanel/AppealsListFilterPanelItem.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement } from 'react';
+import ArrowForwardIos from '@mui/icons-material/ArrowForwardIos';
+import {
+  Box,
+  ListItem,
+  ListItemText,
+  Skeleton,
+  Typography,
+} from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
+import theme from 'src/theme';
+import { AppealStatusEnum } from '../../AppealsContext/AppealsContext';
+
+const useStyles = makeStyles()(() => ({
+  li: {
+    borderBottom: '1px solid #EBECF1',
```

Comment:
Could we add this color to the theme and use it in `AppealsListFilterPanelButton` also instead of hardcoding it?